frontend/src/pull-card/signatures.tsx

```diff
@@ -33,21 +33,25 @@ function Signatures({pull, title, signatures, required}: SignaturesProps) {
          title={noneToShow ? `No ${title} Required` : ''}
       >
       <Box m="2px" mr={noneToShow ? "2px" : 2}>{title}</Box>
-      {NewAndOldSigs(allSignatures)}
+      {NewAndOldSigs(allSignatures, pull)}
```

Comment:
At this point, might be worth making `NewAndOldSigs` into a real React component.
```suggestion
      <NewAndOldSigs allSignatures={allSignatures} pull={pull}/>
```
and add the curly braces to make `NewAndOldSigs` into an actual component.